source/static/api.json

```diff
@@ -87,22 +90,25 @@
           "user"
         ],
         "summary": "Create a new User.\n",
-        "description": "Create a new User.\n",
+        "description": "Create a new User and an organisation for this user.\n",
```

Comment:
More specifically create an Organization if it doesn't exist, join an Organization if it does.